src/network/networkd-manager.c

```diff
@@ -1369,10 +1370,15 @@ int manager_new(Manager **ret) {
         _cleanup_(manager_freep) Manager *m = NULL;
         int r;
 
-        m = new0(Manager, 1);
+        m = new(Manager, 1);
         if (!m)
                 return -ENOMEM;
 
+        *m = (Manager) {
+                .use_speed_meter = true,
```

Comment:
Hmm, I'm not convinced this should be on by default. Waking up every 10 s do a non-trivial operation doesn't seem nice for battery life. Would it be possible to keep this disabled by default and only enable when someone starts listening for the statistics?

Reply:
Yeah, this is the biggest issue. I don't think we want yet-another thing waking up in the background constantly. I'd very much prefer to have this opt-in and on-demand.